pyproject.toml

```diff
@@ -62,15 +62,28 @@ Documentation = "https://pymc-extras.readthedocs.io/"
 Repository = "https://github.com/pymc-devs/pymc-extras.git"
 Issues = "https://github.com/pymc-devs/pymc-extras/issues"
 
+
 [tool.setuptools.packages.find]
 where = ["."]
 include = ["pymc_extras"]
 exclude = ["tests", "notebooks"]
 # Additional install requirements for running tests
 namespaces = false
 
-[tool.setuptools.dynamic]
-version = {file = "pymc_extras/version.txt"}
+
+[tool.hatch.version]
+source = 'vcs'
+
+[tool.hatch.build.targets.sdist]
+exclude = [
+  '/.github',
+  '/notebooks',
+  '/tests'
```

Comment:
Yay, Hatch!!!

```suggestion
```

Excluding tests from the sdists is controversial. Package maintainers like the ability to run tests to make sure everything's working. It's best to reinclude it if possible.

Reply:
What about docs?